AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/DataStoreModelsVersionTests.swift

```diff
@@ -0,0 +1,127 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class DataStoreModelsVersionTests: XCTestCase {
```

Comment:
usually we have the class name of the test correspond to the class name, so if this is another method on `SQLiteStorageEngineAdapter` then move the tests over to SQLLiteStorageEngineAdapterTests. you can also use `// MARK: <section name>` if the file is getting too big

Reply:
Moved to `SQLiteStorageEngineAdapterTests.swift`